packages/next-server/server/next-server.ts

```diff
@@ -269,6 +269,9 @@ export default class Server {
           if (!pathname) {
             throw new Error('pathname is undefined')
           }
+          if (req.method === 'POST') {
```

Comment:
This isn't the right fix. We can remove it for now since it's already broken and address in a different PR.